pkg/virt-handler/vm.go

```diff
@@ -2390,11 +2390,10 @@ func (d *VirtualMachineController) vmUpdateHelperMigrationSource(origVMI *v1.Vir
 
 	if origVMI.Status.MigrationState.AbortRequested {
 		if origVMI.Status.MigrationState.AbortStatus != v1.MigrationAbortInProgress {
```

Comment:
you already check this in handleMigrationAbort() now

Reply:
Right! Thanks